tools/docker/docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       - "9000:9000"
     environment:
       - EDA_HOST=0.0.0.0
-      - EDA_DATABASE_URL=postgresql+asyncpg://postgres:secret@postgres/ansible_events
+      - EDA_DATABASE_URL=postgresql+asyncpg://postgres:secret@postgres/ansible_rulebook
```

Comment:
The ansible_rulebook is a CLI tool. Should the database name represent the project it's related to?

Reply:
Good point.  What should it be?  eda-server?

Reply:
I think eda-server would work.